crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -17,13 +17,18 @@ pub trait Command: Send + Sync + 'static {
     fn write(self, world: &mut World);
 }
 
-/// A list of commands that modify a [`World`], running at the end of the stage where they
-/// have been invoked.
+/// A list of commands that runs at the end of the stage of the system that called them.
+///
+/// Each command can be used to modify the [`World`] for things like:
+/// * spawning entities
+/// * inserting components on new or existing entities
+/// * inserting resources
+/// * etc.
 ///
 /// # Usage
 ///
-/// `Commands` is a [`SystemParam`](crate::system::SystemParam), therefore it is declared
-/// as a function parameter:
+/// To use it, you can declare it as a function parameter of any system since it implements [`SystemParam`](crate::system::SystemParam).
```

Comment:
```suggestion
/// Commands are almost always used as a [`SystemParam`](crate::system::SystemParam).
```

Reply:
My intention with that was to not make it necessary to know what a SystemParam is because a beginner will most likely end up seeing Commands much earlier than learning about the SystemParam implementation.

Reply:
Ah, SGTM.